saul-examples/src/test/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/EntityRelation/EntityRelationTests.scala

```diff
@@ -86,20 +86,20 @@ class EntityRelationTests extends FlatSpec with Matchers {
     ClassifierUtils.InitializeClassifiers(pairs, cls: _*)
 
     PerConstrainedClassifier.onClassifier.classifier.getLabelLexicon.size() should be(2)
-    PerConstrainedClassifier.onClassifier.classifier.getLexicon.size() should be(84)
+    PerConstrainedClassifier.onClassifier.classifier.getLexicon.size() should be(82)
 
     PerConstrainedClassifier.onClassifier.classifier.asInstanceOf[SparseNetworkLearner].getNetwork.get(0).asInstanceOf[LinearThresholdUnit].getWeightVector.size() should be(0)
 
     ClassifierUtils.TrainClassifiers(1, cls_base)
 
-    PerConstrainedClassifier.onClassifier.classifier.asInstanceOf[SparseNetworkLearner].getNetwork.get(0).asInstanceOf[LinearThresholdUnit].getWeightVector.size() should be(1661)
+    PerConstrainedClassifier.onClassifier.classifier.asInstanceOf[SparseNetworkLearner].getNetwork.get(0).asInstanceOf[LinearThresholdUnit].getWeightVector.size() should be(1610)
 
     val jointTrainIteration = 1
     JointTrainSparseNetwork.train[ConllRelation](
       pairs, cls, jointTrainIteration, init = true
     )
 
-    PerConstrainedClassifier.onClassifier.classifier.asInstanceOf[SparseNetworkLearner].getNetwork.get(0).asInstanceOf[LinearThresholdUnit].getWeightVector.size() should be(50)
+    PerConstrainedClassifier.onClassifier.classifier.asInstanceOf[SparseNetworkLearner].getNetwork.get(0).asInstanceOf[LinearThresholdUnit].getWeightVector.size() should be(48)
 
```

Comment:
why the number of features changed then? is it because of those changes in the properties?